bin/truffleruby

```diff
@@ -93,7 +93,8 @@ if [ $on_graalvm = false ]; then
 
     if [ "$mx_time" -gt "$mvn_time" ]; then
         binary_truffle="$root/mx.imports/binary/truffle/mxbuild"
-        source_truffle="$(dirname "$root")/truffle/mxbuild"
+        source_truffle_suite_py=("$(dirname "$root")"/*/*/"mx.truffle/suite.py")
```

Comment:
I changed it to a search in likely directories: `graal`, `truffle` and `compiler`.